base_geoengine/geo_ir/ir_model.py

```diff
@@ -38,7 +38,7 @@
                      ('MULTIPOLYGON', 'MULTIPOLYGON')]
 
 
-class IrModelField(osv.osv):
+class IrModelField(models.Model):
```

Comment:
Model name is `ir.model.fields` so the filename should be `ir_model_fields.py` to be more consistant